pymc3/tests/test_distributions.py

```diff
@@ -1261,6 +1265,7 @@ def test_gamma_logcdf(self):
             {"alpha": Rplusbig, "beta": Rplusbig},
             lambda value, alpha, beta: sp.gamma.logcdf(value, alpha, scale=1.0 / beta),
             skip_paramdomain_outside_edge_test=True,
+            n_samples=-1,
```

Comment:
passed as expected

Reply:
You mean xfailed as expected?

Reply:
yeah, sorry

Reply:
Just making sure we didn't miss anything :D